### ISSUE TITLE
[Suggestion] Option for tooltips to display all available perks

### ISSUE THREAD

**Issue by VagSmashington** [dasilva333/TowerGhostForDestiny#438]
One of the recent changes implemented was:
-Improved tooltips to only display the selected perks instead of all the available perks

Here is my reasoning for adding an option to display the selected perks OR all the available perks,  or undoing it entirely:

Say I have 4 or 5 different purple boots and chest pieces on a character. I want to quickly find which ones allow me to carry more sniper / rocket / MG / shotgun ammo depending if I'm raiding or PVP or whatever. Before, I could use TowerGhost and hover over each one to see which available perks they had. This was much faster than now having to select each piece of armor in the game and having to move the cursor over each perk. 

If I get a new piece of armor and it has the selectable "carry more ammo for..." perk, maybe I want to check if I already have comparable armor with the same perk. With the update I can't easily do that anymore through TowerGhost. I can't even see which perks this armor has without unlocking it and purchasing it before it shows up in TowerGhost. 

3.4.3 (firefox) also has an update which correctly displays the perks for ghosts now. Same idea here, I can now use TowerGhost to quickly mouse over all the ghosts I have to find a good one for relic iron or spinmetal, etc. 

TL;DR - Displaying all armor perks available makes it easier and faster to find armor with specific perk you need.

**Comment by dasilva333** [comment]
I'll consider making it an option in the future, it's not an easy change because it messes with the Missing Perks filter under Progress. Thanks for the feedback

**Comment by slansing97** [comment]
Just my $0.02: I came to the issues page to request this feature back.  Not sure why it was removed, but it's my favorite feature in Tower Ghost.  I was able to search for "sniper" and find out which chest/legs had the perk available so I could start leveling the item.  I would do the same for "void armor".  It also gave you a good preview of items in the "Lost Items" area, the items are now blank.

If this is related to the missing perks change, maybe we can find a solution.

**Comment by dasilva333** [comment]
The slack group is still in effect, my goal is to make the search box allow searching of all perks not just the active ones, also allow an option to show inactive perks in the tooltips. This enhancement isn't a priority at the moment so don't expect these changes soon. Thanks guys

**Comment by dasilva333** [comment]
@VagSmashington Option added to 3.4.8 I called it 'Advanced Tooltips' which:
-shows all perks on the item
-shows aim assist and equip speed too
Please confirm thanks guys

**Comment by VagSmashington** [comment]
Great! Seems to work as you said, thank you very much!.  One thing I noticed:
-The text for aim assist and equip speed overlap are displayed on top of each other
-The perks aren't shown for lost items

I have screen shot examples but for some reason cannot upload them at the moment. Will try again later.

Thanks again
